features/step_definitions/basic_steps.rb

```diff
@@ -1 +1,15 @@
-# put step definitions in this file
\ No newline at end of file
+When(/^I visit the site$/) do
+  visit '/'
+end
+
+Then(/^I should see "([^"]*)" is a student at Craft Academy$/) do |arg1|
+  expect(page).to have_content
+end
+
+When(/^I visit the name page for "([^"]*)"$/) do |arg1|
+  visit '/name/:Johan'
+end
+
+Then(/^i should create a new user named "([^"]*)"$/) do |arg1|
+  expect(page).to have_content 'Johan'
+end
```

Comment:
You are storing a name inside the variable `arg1` and then hard-coding `'Johan'`. That's a big no-no.